docs/configuration/config-file-reference.md

```diff
@@ -5364,6 +5373,36 @@ time_window:
   # lookback delta) that the query should be within. If set to 0, it won't be
   # checked.
   [end: <int> | default = 0]
+
+# Limit that query time range should be within. If not set, it won't be checked.
```

Comment:
Can we elaborate more on `should be within`? If it is within the range it will be rejected or outside the range will be rejected?

Reply:
the ones within the range will be rejected. it `should be within` this range to match the query, and matched queries will be rejected, if this attribute was used in query_rejection.

Reply:
Thanks. Can you document it better? At least when I read it I was confused.

Reply:
Changed the documentation, but didn't specify what will be rejected. Instead specified that query will match if it's within this limit, because same config can be used in priority config and in this case, action will be different. This configuration is only decides whether given query matches the specified requirements in QueryAttributes or not. Action will be decided where it was used(query_rejection or query_priority).